src/test/credentials/auth.test.ts

```diff
@@ -360,6 +365,50 @@ describe('Auth', function () {
         })
     })
 
+    describe('Shared ini files', function () {
+        let tmpDir: string
+
+        beforeEach(async function () {
+            tmpDir = await makeTemporaryToolkitFolder()
+            sinon.stub(SystemUtilities, 'getHomeDirectory').returns(tmpDir)
```

Comment:
would it be more robust to temporarily set $HOME?

Reply:
Home directory behavior is dependent on the operating system so we'd need to change `USERPROFILE` too. I think a stub is OK here as we're not really stubbing out much logic anyway